doc/data/messages/t/too-many-locals/bad.py

```diff
@@ -0,0 +1,11 @@
+def test_addition(num1, num2, num3, num4, num5, num6, num7, num8): # [too-many-locals]
```

Comment:
I think it could be something with two entity with attribute that can become a class later. Like a man and it's animal, you have man_name, man_age, man_surname, man_height, animal_age, animal_cuteness_level, etc. Then in the good example you gave an Animal class with multiple attribute and a Human class. It's close to a real world reason for having too many locals imo.

Reply:
Fixed